control-plane/subcommand/gateway-resources/command.go

```diff
@@ -331,37 +359,104 @@ func (c *Command) loadConfig() error {
 	return nil
 }
 
+// loadGatewayConfigs reads and loads the configs from `/consul/config/config.yaml`, if this file does not exist nothing is done
```

Comment:
To please the linter that's currently failing

```suggestion
// loadGatewayConfigs reads and loads the configs from `/consul/config/config.yaml`, if this file does not exist nothing is done.
```